data/glossary.csv

```diff
@@ -241,7 +218,30 @@ Chaque programme devrait avoir une méthode d’intervention principale. Toutefo
 "IFORM_STATOA","Institutional Forms: Departmental agencies","Structure institutionnelle : Organismes ministériels","Departmental agencies refer to entities with more narrowly defined mandates than ministerial departments, generally specified in the constituent Acts or other instruments. Their specific functions vary widely but tend to be operational in nature, and they usually operate at a distance from government with varying degrees of autonomy. These agencies include organizations listed in Schedule I.1.","Les organismes ministériels sont des entités ministérielles dont le mandat est défini plus étroitement que celui des ministères, qui est généralement précisé dans leur loi constitutive ou d’autres instruments. Leurs fonctions précises varient beaucoup, mais sont généralement de nature opérationnelle. En général, les organismes mènent leurs activités à une certaine distance du gouvernement et leur degré d’autonomie varie considérablement. Ces organismes inclus"
 "IF_IGOC","Institutional Form","Structure institutionnelle","A term used to classify different types of federal organizations and interests.","Expression utilisée pour classifier différents types d'organisations et d'intérêts fédéraux."
 "INACT_PEOPLE","Inactive employees","Employés inactifs","Inactive employees are employees who have been hired to the Public Service but who are not currently remunerated. The employee may be on leave without pay for various reasons (such as Parental Leave, Sick Leave-Without-Pay, Income-Averaging leave, etc.) or their pay may have been stopped for other reasons such as Seasonal Layoffs, or Seasonal layoff periods (i.e. House of Commons, Off-duty status, and unpaid surplus status (12 months)).","Par « employés inactifs », on entend les employés embauchés à la fonction publique qui ne touchent pas de rémunération pour le moment. Un employé peut être en congé non payé pour différentes raisons (congé parental, congé de maladie non payé, congé avec étalement du revenu, etc.) ou sa rémunération peut avoir été suspendue pour d'autres raisons telles que les mises à pied saisonnières ou les périodes de mises à pied saisonnière (à la Chambre des communes, statut d'employé inactif ou statut d'employé excédentaire non rémunéré (12 mois))."
-"INDET_PEOPLE","Indeterminate Employee","Employé nommé pour une période indéterminée","Indeterminate / Permanent employees are persons appointed to the Public Service whose tenure in the position is of an unspecified duration.","Par « employé permanent ou nommé pour une période indéterminée », on entend toute personne nommée à un poste de la fonction publique pour une période indéfinie."
+"INDET_PEOPLE","Indeterminate Employee","Employé nommé pour une période indéterminée","""Employee Type refers to the nature of the appointment to the Public Service. There are four employee types: casual, student, term, indeterminate (including seasonal indeterminate).  
```

Comment:
There's a triple quotation mark here. Is that intended/going to mess with things?

Reply:
Oops, thought I'd fixed that in the source. I'll hotfix it before merging here and double check upstream when I'm in the office.

Reply:
:+1:

Reply:
Oh, wait, I think pipeline didn't pull in the current glossary, this is a version before my latest corrections. I'll just drop the glossary changes from this PR and remind pipeline to make sure they grab a newer glossary source before the next run (thought maybe they always did or had automated that, but maybe not).